src/sentry/static/sentry/app/actionCreators/tags.jsx

```diff
@@ -60,6 +61,34 @@ function tagFetchSuccess(tags) {
   TagActions.loadTagsSuccess(trimmedTags);
 }
 
+export function loadOrganizationTags(api, orgId, selection) {
```

Comment:
The `fetchOrganizationTags` function in this file has a TODO to add more global selection support. This might be a good time to do that over making new functions.

Reply:
This function was meant to replace `fetchOrganizationTags`. At the moment, I'm facing issues with stale values (i.e. out of sync from the `location` object) from the global selection store being improperly propagated to React components. This incurs an additional unnecessary API call 😞 . 

I'm going to try a different approach of: 

1. Have `tagStore` listen to `globalSelectionStore` directly (yay flux)
2. Create `withOrganizationTags` HoC that re-renders whenever `tagStore` is updated
3. Refactor out `fetchOrganizationTags`/`loadOrganizationTags` from `sentry`; with `withOrganizationTags` as the replacement.